auth0/src/main/java/com/auth0/android/provider/OAuthManager.java

```diff
@@ -242,7 +242,15 @@ public void onSuccess(SignatureVerifier signatureVerifier) {
                     validationCallback.onFailure(exc);
                 }
             }
-        });
+        };
+
+        String tokenAlg = decodedIdToken.getHeader().get("alg");
+        if (account.isOIDCConformant() || "RS256".equals(tokenAlg)) {
```

Comment:
When flag is set, RS256 signature verifier will be used. Or if the algorithm in the header states is RS256

Reply:
It would be desired but I don't think we have other SDKs with a flag like this. This flag can be taken as "the expected algorithm is RS256" (because that's what happens on the dashboard when you turn it on).